src/db/migrations/20190403212300-create-user.js

```diff
@@ -30,29 +31,32 @@ export default {
       allowNull: false,
     },
     bio: {
-      type: Sequelize.STRING,
+      type: Sequelize.TEXT,
+      allowNull: true,
+    },
+    interests: {
+      type: Sequelize.ARRAY(Sequelize.TEXT),
       allowNull: true,
     },
     image: {
-      type: Sequelize.STRING,
+      type: Sequelize.TEXT,
       allowNull: true,
     },
-    isActive: {
+    confirmed: {
```

Comment:
we can rename `confirmed` to `isEmailVerified`

Reply:
I have updated it